orchestra/models/core/models.py

```diff
@@ -187,39 +250,14 @@ class Step(StepMixin, models.Model):
     review_policy = JSONField(default={})
     creation_policy = JSONField(default={})
     user_interface = JSONField(default={})
+    todolist_templates_to_apply = models.ManyToManyField(TodoListTemplate,
+                                                         blank=True)
 
     class Meta:
         app_label = 'orchestra'
         unique_together = (('workflow_version', 'slug'),)
 
 
-class Worker(WorkerMixin, models.Model):
```

Comment:
nitpick: no need to rearrange these due to dependencies--- you can make the M2M field reference `'TodoListTemplate'` instead of `TodoListTemplate` and be all set :)